docs/how-to-articles/how-to-simulate-networks-with-netem.md

```diff
@@ -0,0 +1,120 @@
+# Using NetEm to emulate networks
+
+[NetEm](https://www.linux.org/docs/man8/tc-netem.html) (Network Emulator) is an enhancement of the Linux traffic control facilities that allow to add delay, packet loss, duplication and more other characteristics to packets outgoing from a selected  network  interface.  NetEm  is  built using  the  existing  Quality  Of Service (QOS) and Differentiated Services (diffserv) facilities in the Linux kernel.
+
+It's a great tool to test SRT connections with network impairments like on real networks.
+
+## Installation 
+
+The Linux traffic control (tc) and NetEm (netem) are available in most linux distributions and typically part of the `iproute2` package.
+
+On Ubuntu systems can be installed by following command:
+
+```
+sudo apt-get install iproute2
+```
+
+## Installation on CentOS 8.2
+
+On CentOS 8 it's part of `iproute-tc`
+
+```
+sudo yum install iproute-tc
+```
+
+RHEL based systems like CentOS 8.2 might need an additional Kernel module, if the look-up of the name for `qdisc` fails, like in example shown below:
+
+```
+sudo tc qdisc add dev enp7s0 root netem delay 10ms 10ms loss 2% rate 12mbit
+Error: Specified qdisc not found.
+```
+
+The modules `kernel-modules-extra` and `kernel-debug-modules-extra` can be installed with following command:
+
+```
+sudo yum install kernel-debug-modules-extra kernel-modules-extra
+```
+
+After installation a reboot is needed to activate the kernel modules.
```

Comment:
```suggestion
**NOTE**: RHEL-based systems like CentOS 8.2 might need an additional kernel module in order to run netem. This would be indicated if, in running netem, the look-up of the name for `qdisc` fails, like in example shown below:

```
sudo tc qdisc add dev enp7s0 root netem delay 10ms 10ms loss 2% rate 12mbit
Error: Specified qdisc not found.
```

In such a case, install the modules `kernel-modules-extra` and `kernel-debug-modules-extra` with following command:

```
sudo yum install kernel-debug-modules-extra kernel-modules-extra
```

After installation a reboot is needed to activate the kernel modules.
```